Now with Lit Review

JoePrezioso opened the pull request:
As requested: fixed! :)

Commit:
Now with Lit Review

As requested: fixed! :)

decause commented:
Close.

Since the .yaml files are being parsed by a script, and then used to populate templates that are auto-generated, there is a certain amount of fragility built into the system. This means that naming and whitespace are _super_ important.

As per our discussions in class, the correct form for those assignment names in your .yaml file is:

litreview1
quiz1

Once you've changed the names, I'll merge.

Commit:
Removed the whitespace

Okay, so...proper formatting is important. I wasn't aware that the system was that fragile.

decause commented:
:100:
